src/main/webapp/app/admin/legal/legal-document-update.component.html

```diff
@@ -1,44 +1,44 @@
             [resizableMaxHeight]="maxHeight"
             [enableResize]="true"
+            [enableFileUpload]="false"
             [minHeightEditor]="700"
             class="markdown-editor"
         ></jhi-markdown-editor>
-        <div class="instructions-status">
+        <div>
             <div *ngIf="!unsavedChanges && !isSaving">
-                <fa-icon [icon]="faCheckCircle" class="text-success me-1" ngbTooltip="{{ 'artemisApp.privacyStatement.savedTooltip' | artemisTranslate }}"></fa-icon>
-                <span class="text-info" jhiTranslate="artemisApp.privacyStatement.saved">Saved.</span>
+                <fa-icon [icon]="faCheckCircle" class="text-success me-1" ngbTooltip="{{ 'artemisApp.legal.savedTooltip' | artemisTranslate }}"></fa-icon>
+                <span class="text-info" jhiTranslate="artemisApp.legal.saved">Saved.</span>
             </div>
             <div *ngIf="unsavedChanges && !isSaving">
-                <fa-icon [icon]="faExclamationTriangle" class="text-warning me-1" ngbTooltip="{{ 'artemisApp.privacyStatement.unsavedTooltip' | artemisTranslate }}"></fa-icon>
-                <span class="text-info" jhiTranslate="artemisApp.privacyStatement.unsaved">Unsaved.</span>
+                <fa-icon [icon]="faExclamationTriangle" class="text-warning me-1" ngbTooltip="{{ 'artemisApp.legal.unsavedTooltip' | artemisTranslate }}"></fa-icon>
+                <span class="text-info" jhiTranslate="artemisApp.legal.unsaved">Unsaved.</span>
             </div>
             <div *ngIf="isSaving">
-                <fa-icon [icon]="faCircleNotch" class="me-1" ngbTooltip="{{ 'artemisApp.privacyStatement.savingTooltip' | artemisTranslate }}"></fa-icon>
-                <span class="text-info" jhiTranslate="artemisApp.privacyStatement.saving">Saving...</span>
+                <fa-icon [icon]="faCircleNotch" class="me-1" ngbTooltip="{{ 'artemisApp.legal.savingTooltip' | artemisTranslate }}"></fa-icon>
+                <span class="text-info" jhiTranslate="artemisApp.legal.saving">Saving...</span>
             </div>
         </div>
         <div class="btn-below-editor">
-            <jhi-button
-                id="update-privacy-statement-btn"
-                (click)="updatePrivacyStatement()"
-                [isLoading]="isSaving"
-                [disabled]="!markdownEditor?.markdown || markdownEditor?.markdown?.trim() === ''"
-                [title]="'entity.action.save'"
-                [icon]="faSave"
-                >Save
-            </jhi-button>
+            <button
+                type="submit"
+                (click)="updateLegalDocument()"
+                class="btn btn-primary"
+                id="update-legal-document-btn"
+                [disabled]="!markdownEditor?.markdown || markdownEditor?.markdown?.trim() === '' || isSaving"
+            >
+                <fa-icon [icon]="faSave"></fa-icon>
+                <span jhiTranslate="entity.action.save">Save</span>
+            </button>
```

Comment:
Why switch from `jhi-button` to `button`?

Reply:
jhi-button had the weird behavior that it was clickable although disabled. Using the button was an easy fix here